src/frontend/src/flows/manage.ts

```diff
@@ -50,6 +55,58 @@ export const renderManage = (
 const init = async (userNumber: bigint, connection: IIConnection) => {
   // TODO - Check alias for current identity, and populate #nameSpan
   initLogout();
+  const addAdditionalDevice = document.querySelector(
+    "#addAdditionalDevice"
+  ) as HTMLButtonElement;
+
+  addAdditionalDevice.onclick = async () => {
+    let newDevice: WebAuthnIdentity;
+    try {
+      newDevice = await WebAuthnIdentity.create();
+    } catch (error) {
+      await displayError({
+        title: "Failed to add new device",
+        message: html`
+          We failed to add your new device.<br />
+          If you're trying to add a device that is not attached to this machine
+          try following the instructions at<br />
+          <a
+            target="_blank"
+            href="https://sdk.dfinity.org/docs/ic-identity-guide/auth-how-to.html#_add_a_device"
+            >https://sdk.dfinity.org/docs/ic-identity-guide/auth-how-to.html#_add_a_device</a
+          >
+        `,
+        detail: error.message,
+        primaryButton: "Try again",
+      });
+      return renderManage(userNumber, connection);
+    }
+    const deviceName = await pickDeviceAlias();
+    if (deviceName === null) {
+      return renderManage(userNumber, connection);
+    }
+    // TODO check whether newDevice is already registered
```

Comment:
```suggestion
    // TODO check whether newDevice is already registered,
    // or better, pass existing devices to `.create` so that they
    // cannot be added again
```